src/server/server.cpp

```diff
@@ -590,14 +590,17 @@ void server::read_version(socket_ptr socket, std::shared_ptr<simple_wml::documen
 		const simple_wml::string_span& version_str_span = (*version)["version"];
 		const std::string version_str(version_str_span.begin(), version_str_span.end());
 
+		const simple_wml::string_span& source_str_span = (*version)["client_source"]; // TODO: handle this attribute not being present on previous versions of the client
```

Comment:
This will just be empty for old clients which seems fine, no?

Reply:
Ah, yeah, I had tested that but apparently forgot to remove to TODO comment I left myself.